apps/test/unit/templates/manageStudents/ManageStudentsTableTest.js

```diff
@@ -354,5 +357,176 @@ describe('ManageStudentsTable', () => {
+            <Provider store={getStore()}>
+              <ManageStudentsTable section={wordSection} />
+            </Provider>
+          );
+
+          expect(
+            wrapper.containsMatchingElement(
+              <ManageStudentsNotificationFull
+                manageStatus={{
+                  ...defaultAddTransferStatus,
+                  status: 'full',
+                  type: null,
+                  error: null,
+                  sectionDisplay: '',
+                  verb: 'copy'
+                }}
+              />
+            )
+          ).to.be.true;
+        });
+      });
+      describe('Multiple students', () => {
+        const defaultAddTransferStatus = {
+          sectionCapacity: 500,
+          sectionCode: 'ABCDEF',
+          sectionStudentCount: 499,
+          numStudents: 2
+        };
+
+        it('added', () => {
+          getStore().dispatch(
+            addStudentsFull(defaultAddTransferStatus, [0, 1])
+          );
+
+          const wrapper = mount(
+            <Provider store={getStore()}>
+              <ManageStudentsTable section={wordSection} />
+            </Provider>
+          );
+
+          expect(
+            wrapper.containsMatchingElement(
+              <ManageStudentsNotificationFull
+                manageStatus={{...defaultAddTransferStatus, status: 'full'}}
+              />
+            )
+          ).to.be.true;
+        });
+        it('moved', () => {
+          getStore().dispatch(
+            transferStudentsFull(defaultAddTransferStatus, false)
+          );
+
+          const wrapper = mount(
+            <Provider store={getStore()}>
+              <ManageStudentsTable section={wordSection} />
+            </Provider>
+          );
+
+          expect(
+            wrapper.containsMatchingElement(
+              <ManageStudentsNotificationFull
+                manageStatus={{
+                  ...defaultAddTransferStatus,
+                  status: 'full',
+                  type: null,
+                  error: null,
+                  sectionDisplay: '',
+                  verb: 'move'
+                }}
+              />
+            )
+          ).to.be.true;
+        });
+        it('copied', () => {
+          getStore().dispatch(
+            transferStudentsFull(defaultAddTransferStatus, true)
+          );
+
+          const wrapper = mount(
+            <Provider store={getStore()}>
+              <ManageStudentsTable section={wordSection} />
+            </Provider>
+          );
+
+          expect(
+            wrapper.containsMatchingElement(
+              <ManageStudentsNotificationFull
+                manageStatus={{
+                  ...defaultAddTransferStatus,
+                  status: 'full',
+                  type: null,
+                  error: null,
+                  sectionDisplay: '',
+                  verb: 'copy'
+                }}
+              />
+            )
+          ).to.be.true;
+        });
+      });
+    });
```

Comment:
nit: can we also add a test to make sure the alert doesn't show when it's not supposed to?

Reply:
I've added these tests as well, but it gets a bit wordy. Happy to refactor/redact some of this if it feels a bit too much here!